.dockerignore

```diff
@@ -2,3 +2,5 @@ node-feature-discovery
 node-feature-discovery-job.json
 vendor/
 demo/
+.dockerignore
+.git*
```

Comment:
I think you can remove some files from it (like `node-feature-discovery`) and maybe sort it.

Reply:
Thanks for looking at this. PR updated (sorted and the outdated entries removed/updated)